dist/stat_data.py

```diff
@@ -464,6 +464,15 @@ def __init__(self, name, desc, flags=''):
     YieldStat('page_locked_blocked', 'page acquire locked blocked'),
     YieldStat('page_read_blocked', 'page acquire read blocked'),
     YieldStat('page_sleep', 'page acquire time sleeping (usecs)'),
```

Comment:
The statistics should be sorted alphabetically by their symbol in this file.

Some of the descriptions have the word capitalised, others don't. Usage should be consistent with other places.

There are also spaces at the start of some description strings